pkg/psmdb/pmm.go

```diff
@@ -293,11 +293,13 @@ func pmmAgentEnvs(spec api.PMMSpec, secret *corev1.Secret, customLogin bool, cus
 }
 
 func PMMAgentScript(cr *api.PerconaServerMongoDB) []corev1.EnvVar {
+	// handle disabled TLS
+
 	pmmServerArgs := "$(PMM_ADMIN_CUSTOM_PARAMS) --skip-connection-check --metrics-mode=push "
 	pmmServerArgs += " --username=$(DB_USER) --password=$(DB_PASSWORD) --cluster=$(CLUSTER_NAME) "
 	pmmServerArgs += "--service-name=$(PMM_AGENT_SETUP_NODE_NAME) --host=$(DB_HOST) --port=$(DB_PORT)"
 
-	if cr.CompareVersion("1.13.0") >= 0 && !cr.Spec.UnsafeConf {
+	if cr.CompareVersion("1.13.0") >= 0 {
```

Comment:
Why it was removed?

Reply:
good catch, we need to check if tls enabled there: https://github.com/percona/percona-server-mongodb-operator/pull/1504/commits/6badccfc800b3b7cd469a4a1bdecc9223eb4ab54